packages/near-membrane-shared/src/ArrayBuffer.ts

```diff
@@ -0,0 +1,8 @@
+import { ObjectLookupOwnGetter } from './Object';
+
+const ArrayBufferCtor = ArrayBuffer;
```

Comment:
Why is this called Ctor btw. Is it not `ArrayBuffer.constructor`, why can't we just use ArrayBuffer then? I guess the same with Date, etc. Like we don't do it for Boolean, so I'm just confused when we do this.

Reply:
`ArrayBuffer` is `ArrayBuffer.prototype.constructor`

Reply:
We do it any time we store a reference to the Capital Letter constructor and reuse it. See `ArrayCtor`, `ProxyCtor`, etc.

Reply:
Ah I see. When extracted to this repo the reuse no longer applies so I can clean that up. Fixed.